packaging/ios/Subsurface-mobile.pro

```diff
@@ -104,7 +104,10 @@ SOURCES += ../../subsurface-mobile-main.cpp \
 	../../profile-widget/diverectitem.cpp \
 	../../profile-widget/divetextitem.cpp
 
-RESOURCES += ./qml.qrc ../../mobile-widgets/qml/mobile-resources.qrc ./translations.qrc
+RESOURCES += ./qml.qrc \
+			../../mobile-widgets/qml/mobile-resources.qrc \
+			../../map-widget/qml/mapWidget.qrc \
```

Comment:
the filename is `MapWidget.qrc` but here `mapWidget.qrc` is used, would that be a problem?

also, why not name the file lowercase as per the Qt standard? `map-widget.qrc`

Reply:
yeah, I'd prefer all lowercase...
IIRC Mac still allows for a filesystem to not be case sensitive.